### PR TITLE
Added delete comment

### PR DESCRIPTION
**Pull request by zaneris123**

### CODE REVIEW

File: controllers/comments.controller.js [hunk 1]
```diff
@@ -19,4 +19,14 @@ exports.postComments = (req, res, next) => {
     .catch((err)=>{
         next(err)
     })
+}
+
+exports.deleteComments = (req, res, next) => {
+    removeComments(req.params.id)
+    .then(()=>{
+        res.status(204).send("")
```
Comment (pippaamy) [inline controllers/comments.controller.js:27]: we don't need to have the .send("") here :)

**Review by pippaamy — COMMENTED** [review]
Great work! just a couple of changes to make!

File: __tests__/api.test.js [hunk 2]
```diff
@@ -347,4 +347,20 @@ describe("PATCH: updated reviews votes count",()=>{
...
+    })
+    test("400: Invalid comment entry",()=>{
+        return request(app)
+        .delete("/api/comments/dskad")
+        .expect(400)
+        .then(({body})=>{
+            expect(body).toEqual({msg: "Invalid entry"})
+        })
+    })
+})
```
Comment (johnhutcheon) [inline __tests__/api.test.js:366]: Can we add another test here please? a 404 for a non-existent comment id?

File: models/comments.model.js [hunk 3]
```diff
@@ -18,4 +18,9 @@ exports.insertCommentById = (bodyObj, reviewId) => {
             .then((data)=>{
                     return data.rows[0]
             })
+}
+
+exports.removeComments = (commentId) => {
+    return db.query(`DELETE FROM comments WHERE comment_id = $1;`,[commentId])
+        .then(()=>{return})
```
Comment (johnhutcheon) [inline models/comments.model.js:25]: We'll need to add a 404 error handler here for a non-existent ID, to pass the test I've mentioned in my other comment 👍

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Hi Zain, can you just see my comments please and add some code/testing for a non existent ID please? Thanks!

**Pull request merged**